integration/src/test/java/org/mule/test/routing/UntilSuccessfulTestCase.java

```diff
@@ -193,6 +230,22 @@ public void untilSuccessfulInErrorHandler() throws Exception {
     assertThat(event.getMessage().getPayload().getValue(), is("hello"));
   }
 
+  @Test
+  public void concurrentEventsWithLimitedCores() throws Exception {
+    for (int i = 0; i < 3; i++) {
+      new Thread(() -> {
```

Comment:
usa en `Executor` instead of a `Thread`

Reply:
This test was experimental. Just removed it.